tests/integration/utils/fixture.mjs

```diff
@@ -1,8 +1,9 @@
 // @ts-check
 import { rm } from 'fs/promises'
+import { join } from 'path'
 import { fileURLToPath } from 'url'
 
-import cpy from 'cpy'
+import { copy } from 'fs-extra'
```

Comment:
`cpy` cannot copy symlinks, so I exchanged it for `fs-extra -> copy`

Reply:
whats about `fs.cp` with `recursive` option? I think this uses the file system copy under the hood which should copy symlinks as well

Reply:
Only available as of Node.js 16.7.0